src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings, or other styles with `*` is still inefficient for the browser.